doc/source/whatsnew/v1.4.0.rst

```diff
@@ -637,8 +637,8 @@ Timedelta
 Timezones
 ^^^^^^^^^
 - Bug in :func:`to_datetime` with ``infer_datetime_format=True`` failing to parse zero UTC offset (``Z``) correctly (:issue:`41047`)
+- Clarified :func:`to_datetime` documentation concerning parameter ``utc`` and the impact of its default value (``False``) on parsing datetimes from a timezone with varying time offsets (daylight savings) (:issue:`42229`).
```

Comment:
We don't add whatsnew entries for doc changes anymore.

Reply:
ok